### PR TITLE
Added announce. and favWorkflow presenter Tests and Bug Fixes in Announcements

### PR DESCRIPTION
**Pull request by Hiteshgautam01**
Please make sure these boxes are checked before submitting your pull request - thanks!

- [x] Apply the `AndroidStyle.xml` style template to your code in Android Studio.

- [x] Run the checks with `./gradlew check` to make sure you didn't break anything

- [x] If you have multiple commits please combine them into one commit by squashing them.

### CODE REVIEW

File: app/src/main/java/org/apache/taverna/mobile/ui/anouncements/AnnouncementPresenter.java [hunk 1]
```diff
@@ -62,7 +62,6 @@ public void detachView() {
 
     public void loadAllAnnouncement(int pageNumber) {
         checkViewAttached();
-        getMvpView().showProgressbar(true);
```
Comment (sagar15795) [inline app/src/main/java/org/apache/taverna/mobile/ui/anouncements/AnnouncementPresenter.java]: why have you removed this line?

Comment (Hiteshgautam01) [reply app/src/main/java/org/apache/taverna/mobile/ui/anouncements/AnnouncementPresenter.java]: While announcements are loading, due to this we can see two progress bar at the same time. You can see in this video:- https://drive.google.com/file/d/1GaAdVFM7PGhusLZizwuAo2rIBD2Zj4ly/view?usp=sharing
So, I removed one progress bar in loadAllAnnouncement.

Comment (sagar15795) [reply app/src/main/java/org/apache/taverna/mobile/ui/anouncements/AnnouncementPresenter.java]: Nice!

**Comment by sagar15795** [comment]
@Hiteshgautam01 Great work. Just make sure you will test `attachSearchHandler` function in instrumental testing.

**Pull request merged**